draco/learn.py

```diff
@@ -0,0 +1,231 @@
+import logging
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from matplotlib.colors import ListedColormap
+from sklearn import svm
+from sklearn.decomposition import PCA
+from sklearn.model_selection import train_test_split
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class DracoLearn:
```

Comment:
I don't know whether this needs to be a class vs just a collection of methods. Is there benefit to having a class?